### PR TITLE
Sfl 256

### PR DESCRIPTION
**Pull request by vineet-webonise**

### CODE REVIEW

File: SUNFLOWR/SUNFLOWR/MindVC/Mind+TableView.swift [hunk 1]
```diff
@@ -60,7 +60,20 @@ extension MindVC: UITableViewDelegate, UITableViewDataSource {
     }
     
     @objc func startVideo(sender: UIButton) {
-        playVideo(row: sender.tag)
+        switch sender.titleLabel?.text {
+        case Constants.ActivityName.Swap:
+            setDataForSwapApiCall(row: sender.tag)
```
Comment (saketPandhare-Webonise) [inline SUNFLOWR/SUNFLOWR/MindVC/Mind+TableView.swift:65]: Move "Start" to constant like swap

File: SUNFLOWR/SUNFLOWR/MoveVC/Move+TableView.swift [hunk 2]
```diff
@@ -60,13 +60,25 @@ extension MoveVC: UITableViewDelegate, UITableViewDataSource {
...
     
-    ///ToDO:
     @objc func dropDownBtnTapped(sender: UIButton) {
         expandCollapseCellAction(btnTag: sender.tag)
     }
     
-    @objc func startVideo(sender: UIButton) {
-        playVideo(row: sender.tag)
+   @objc func startVideo(sender: UIButton) {
+        switch sender.titleLabel?.text {
```
Comment (saketPandhare-Webonise) [inline SUNFLOWR/SUNFLOWR/MoveVC/Move+TableView.swift:68]: same here

Comment (vineet-webonise) [reply SUNFLOWR/SUNFLOWR/MoveVC/Move+TableView.swift:68]: Done

File: SUNFLOWR/SUNFLOWR/MindVC/Mind+TableView.swift (hunk 1 above)
Comment (vineet-webonise) [reply SUNFLOWR/SUNFLOWR/MindVC/Mind+TableView.swift:65]: Done

**Pull request merged**